peps/pep-9999.rst

```diff
@@ -0,0 +1,292 @@
+
+Vote of no confidence
+---------------------
+
+In exceptional circumstances, the Packaging community may remove a sitting council member, or the entire council, via a vote of no confidence.
+
+A no-confidence vote is triggered when a Packaging community member calls for one publicly on an appropriate public communication channel, and another Packaging community member seconds the call within two weeks.
+
+The vote lasts for two weeks. Packaging community members vote for or against. If at least two thirds of voters express a lack of confidence, then the vote succeeds. Quorum for a vote of no confidence is 50%.
+
+There are two forms of no-confidence votes: those targeting a single member, and those targeting the council as a whole. The initial call for a no-confidence vote must specify which type is intended. If a single-member vote succeeds, then that member is removed from the council and the resulting vacancy can be handled in the usual way. If a whole-council vote succeeds, the council is dissolved and a new council election is triggered immediately.
+
+-----------------------
+Changing the governance
+-----------------------
+
+Changes to this governance model, once it is accepted, will require at least a two-thirds majority of votes cast in a Packaging community vote which should be open for two weeks.
+
+==============
+Rejected Ideas
+==============
+
+----------------------------------------
+Annual elections for all council members
+----------------------------------------
+
+An annual term for council members is the approach taken for the Python steering council's elections. This PEP uses a cohort-based model, derived from the PSF Board's elections which enables continuity of members across a changing council.
+
+There is a trade-off between continuity of the council and reshuffles. This PEP takes the position that continuity will be more valuable for the Python Packaging space, especially combined with the vote of no confidence, automatic removal of inactive voters and regular elections.
+
+-------------------------------
+Term limits for council members
+-------------------------------
+
+While this is viewed as valuable for boards in general, this was rejected because of the size of the pool of interested and qualified people who might serve.
+
+-------------------------------
+Approval voting in the election
+-------------------------------
+
+An earlier non-public draft of this PEP utilised an approval voting process, which aligned with what :pep:`13` stated at the time of writing. Since the Python core team has consensus to move away from that and utilise BLOC-STAR, this PEP was updated to align with that for the same reasons as the core teams' move to BLOC STAR: it better captures voter intention in the results.
+
```

Comment:
The move to BLOC-STAR for the Steering Council has occurred, so this should be rephrased to mirror that development.